src/compute/src/server.rs

```diff
@@ -525,8 +525,6 @@ impl<'w, A: Allocate> Worker<'w, A> {
             let mut old_compaction = BTreeMap::default();
             // Exported identifiers from dataflows we retain.
             let mut retain_ids = BTreeSet::default();
-            // `as_of` frontiers of collections exported from requested dataflows.
-            let mut new_as_ofs = BTreeMap::default();
```

Comment:
I realized that there is no need to have this map, we can just use `old_compaction` instead.